app/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
     alias(libs.plugins.jetbrains.kotlin.android)
     alias(libs.plugins.kotlin.kapt)
     alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.compose.compiler)
```

Comment:
Google's [own migration doc](https://developer.android.com/develop/ui/compose/compiler) says to *also* add a line to the top-level gradle file (in addition to the `app` folder one) - 
```kotlin
    alias(libs.plugins.compose.compiler) apply false
```